app/controllers/financial_planners_controller.rb

```diff
@@ -6,9 +6,9 @@ def new
   def show
     @financial_planner = FinancialPlanner.find(params[:id])
     @status = params[:status]
-    @financial_planner_before_reservations = Reservation.where(fp_id: current_fp.id).where(day: (Date.current)..(Date.current.since(3.month)))
-    @financial_planner_after_reservations = Reservation.where(fp_id: current_fp.id).where("day <= ?", Date.current)
-    @select_fp_schedules = Reservation.all.where(day: (Date.current)..(Date.current.since(3.month))).where(fp_id: current_fp.id).order(day: :desc)
+    @financial_planner_before_reservations = Reservation.where(fp_id: current_financial_planner.id).where(day: (Date.current)..(Date.current.since(3.month)))
+    @financial_planner_after_reservations = Reservation.where(fp_id: current_financial_planner.id).where("day <= ?", Date.current)
+    @select_fp_schedules = Reservation.all.where(day: (Date.current)..(Date.current.since(3.month))).where(fp_id: current_financial_planner.id).order(day: :desc)
```

Comment:
```suggestion
    @select_fp_schedules = Reservation.where(day: (Date.current)..(Date.current.since(3.month))).where(fp_id: current_financial_planner.id).order(day: :desc)
```

Reply:
消します！